group/iab/IAB_Decisions.md

```diff
@@ -54,56 +54,30 @@ for the IAB chair to determine whether it will be held open or secret.
 ## Normal Process
 
 The IAB chair (or executive director on the instructions of the chair) will 
-typically follow the following process:
-
-- If the request for a vote comes from an external source (e.g., the NomCom
-  provides its IESG slate for confirmation), then, as soon as possible after
-  receiving the request, the IAB chair adds the vote topic to the agenda of the
-  next IAB teleconference. If the next meeting is less than 48 hours away, the
-  chair should instead add the topic to the following teleconference.
-
-- The chair emails the list of voting members to announce the date of the
-  discussion, whether ballots are open or secret, and start an email discussion
-  thread on the topic.
-  
-- The agenda of that meeting will include the vote discussion, and needs to be
-  posted at least 48 hours in advance. If the discussion will happen in an
-  irregularly scheduled meeting (i.e. meetings other than the weekly IAB
-  meeting teleconference or meetings prescheduled during IETF week) then, the
-  new meeting needs to be announced at least a week in advance.
-
-- Discussion and votes will continue on the voting members list, and during the
-  teleconference.
-
-- The vote will typically stay open until a vote has been recorded for every
+typically follow the following process to initiate a vote:
+
+- The chair emails the list of voting members to start an email discussion on
+  the topic, stating whether ballots are open or secret.
```

Comment:
Typically we don't announce the ballot being open--it just defaults to that. I wonder if it should say "optionally indicating that the ballot will be secret."?

Reply:
Makes sense.
```suggestion
  the topic, optionally indicating that the ballots will be secret.
```